homeassistant/components/comfoair/fan.py

```diff
@@ -87,30 +90,30 @@ def speed(self):
     @property
     def speed_list(self):
         """List of available fan modes."""
-        return [SPEED_OFF, SPEED_LOW, SPEED_MEDIUM, SPEED_HIGH]
+        return list(SPEED_MAPPING.values())
```

Comment:
Please create a new constant at the module level so we don't need to do an extra call on every access. It's ok to depend on the `SPEED_MAPPING` constant when creating the other constant.